tests/test_app/TestApp/LoadPlugin.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.7
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace TestApp;
+
+use Cake\Core\Configure;
+use Cake\Routing\Router;
+use ReflectionClass;
+
+class LoadPlugin
+{
+    /**
+     * simulate running the Application
+     *
+     * @return \Cake\Http\BaseApplication
+     */
+    public static function load($plugins = [])
+    {
+        try {
```

Comment:
Should we be using PluginCollection::add() here instead? The collection can be accessed from Plugin.  This 'helper' could potentially be added to TestCase. I suspect there are a few apps and plugin test suites that need to load plugins without an application around.